ta4j-core/src/main/java/org/ta4j/core/trading/rules/WaitForRule.java

```diff
@@ -54,8 +54,8 @@ public WaitForRule(OrderType orderType, int numberOfBars) {
 
     @Override
     public boolean isSatisfied(int index, TradingRecord tradingRecord) {
-        boolean satisfied = false;
```

Comment:
I think this was 'more convenient'. Please take a look at my comment under this PR